airgun/views/host_new.py

```diff
@@ -344,6 +344,34 @@ class networking_interfaces(Card):
         class networking_interface(Card):
             pass
 
+        @View.nested
+        class virtualization(Card):
+            ROOT = './/article[contains(@data-ouia-component-id, "card-template-Virtualization")]'
+
+            datacenter = Text('.//div[contains(@class, "pf-c-description-list__group")][1]//dd')
+            cluster = Text('.//div[contains(@class, "pf-c-description-list__group")][2]//dd')
+            memory = Text('.//div[contains(@class, "pf-c-description-list__group")][3]//dd')
+            public_ip_address = Text(
+                './/div[contains(@class, "pf-c-description-list__group")][4]//dd'
+            )
+            mac_address = Text('.//div[contains(@class, "pf-c-description-list__group")][5]//dd')
+            cpus = Text('.//div[contains(@class, "pf-c-description-list__group")][6]//dd')
+            cores_per_socket = Text(
+                './/div[contains(@class, "pf-c-description-list__group")][7]//dd'
+            )
+            firmware = Text('.//div[contains(@class, "pf-c-description-list__group")][8]//dd')
+            hypervisor = Text('.//div[contains(@class, "pf-c-description-list__group")][9]//dd')
```

Comment:
The XPath is very common in all of them (Expected) apart from their integer position value.

Could we simply generate the paths instead of repeating them through looping or any other better mechanism ?  

Not a blocking comment but could shorten the code and easy to maintain.

Reply:
@sambible I think that my PR could be an inspiration on generating locators https://github.com/LadislavVasina1/airgun/blob/70689c1f2597c778df5fab62c9ce33beb89a7b69/airgun/entities/host_new.py#L427-L445